examples/C++/src/README.md

```diff
@@ -155,6 +157,10 @@ manip_image output.hv
 ```
 ${EXE_LOC}/demo2
 ```
+
+As before, input a projection data filename (e.g. `sino.hs`) and image data filename (e.g. `image.hv`) when prompted. 
+Subsequently enter `Matrix` when asked `Which type do you want? Possible values are:` (in reference to backprojector type) and `Ray Tracing` when prompted with `matrix type := `. _Other options may fail as they have deprecated._
+
```

Comment:
This is about the best I could do with `demo2` with minor modifications. The demo is there to demonstrate the prompting STIR has but in this instance, only a few options are valid from the provided list: Interpolation, Matrix, None and Post Smoothing.

Reply:
`parallelproj` would work as well (if it is available), just to make things complicated. not 100% sure how to phrase that. Maybe better to exclude the `Interpolation` one. (Note grammar error in "have deprecated")